docs/DeveloperGuide_Winery.md

```diff
@@ -0,0 +1,622 @@
+This includes node types, relationship types, policy types, artifact types, artifact templates, and
+artifacts such as virtual machine images.
+
+The topology modeler enables to create service templates. Service templates consists of instances of node types
+(node templates) and instances of relationship types (relationship templates). They can be annotated with
+requirements and capabilities, properties, and policies.
+
+The BPMN4TOSCA plan modeler offers web based creation of BPMN models with the TOSCA extension
+BPMN4TOSCA. That means the modeler supports the BPMN elements and structures required by TOSCA plans
+and not the full set of BPMN [KBBL12]. This part is currently in development and not part of the opensourced
+code of Winery.
+
+The repository stores TOSCA models and allows managing their content. For instance, node types, policy types,
+and artifact templates are managed by the repository. The repository is also responsible for importing and 
+exporting CSARs, the exchange format of TOSCA files and related artifacts.
+
+---
+
+#### Code Location
+
+Winery is an Eclipse hosted project. Therefore, the code is versioned using git. One can browse the repository
+at [http://git.eclipse.org/c/winery/org.eclipse.winery.git/](http://git.eclipse.org/c/winery/org.eclipse.winery.git/). In the root of the repository, a README.md
+[http://git.eclipse.org/c/winery/org.eclipse.winery.git/plain/README.md](http://git.eclipse.org/c/winery/org.eclipse.winery.git/plain/README.md) is describing the inital setup in
+Eclipse and how to build the WAR files.
+
+---
+
+#### General Compilation Remarks
+
+Winery is built using Maven. The Javascript libraries are fetched using bower. More information is contained in
+the README.md
+
+---
+
+#### Overview on Projects
+
+This section provides a short overview on all projects Winery consists of.
+
+##### Project org.eclipse.winery.common
+
+This project contains classes used by the repository and topology modeler projects. It contains constants, the id
+system, interface definitions for repository access and the datatype to manage Winery's extended key/value properties.
+
+##### Project org.eclipse.winery.highlevelrestapi
+
+This project contains an API to communicate with REST services. It provides an abstraction layer above Apache
+Commons HTTPClient. It is used by projects generated by the IA generator to upload compiled 
+implementations as implementation artifacts.
+
+##### Project org.eclipse.winery.generators.ia
+
+This project contains the generator which generates a NodeType implementation based on a NodeType
+interface.
+
+##### Project org.eclipse.winery.model.csar.toscametafile
+
+This project contains the model for the file "Tosca.meta".
+
+##### Project org.eclipse.winery.model.selfservice
+
+This project contains the model for the self service portal. It is used by the Vinothek [BBKL14] to display user
```

Comment:
Can you reformat the text to have one sentence per line?

Sentence 1 with words.
Sentence 2 with words.
Sentence 3 with words.
etc